### PR TITLE
Escape xml characters

### PR DESCRIPTION
**Pull request by philliprower**
Escapes XML special characters contained in text body of custom attributes.  Fix errors I encountered using special characters where CAS Service Ticket validation response was invalid XML.

### CODE REVIEW

File: support/cas-server-support-validation/src/main/java/org/apereo/cas/web/view/Cas30ResponseView.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package org.apereo.cas.web.view;
 
+import org.apache.commons.lang.StringEscapeUtils;
```
Comment (mmoayyed) [inline support/cas-server-support-validation/src/main/java/org/apereo/cas/web/view/Cas30ResponseView.java]: Use `org.apache.commons.lang3.StringEscapeUtils;`

Comment (philliprower) [inline support/cas-server-support-validation/src/main/java/org/apereo/cas/web/view/Cas30ResponseView.java]: Ok.

File: support/cas-server-support-validation/src/main/java/org/apereo/cas/web/view/Cas30ResponseView.java [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package org.apereo.cas.web.view;
 
-import org.apache.commons.lang.StringUtils;
```
Comment (philliprower) [inline support/cas-server-support-validation/src/main/java/org/apereo/cas/web/view/Cas30ResponseView.java]: I updated both classes from commons lang to lang3.

Comment (mmoayyed) [inline support/cas-server-support-validation/src/main/java/org/apereo/cas/web/view/Cas30ResponseView.java]: Great! Thank you. 

One other thing you may want to do is merge with master once, and then update the pull request. Right now, you have about 250+ commits in this PR. If you merge with master, the count should go down to one.

Do you know how you might do that? If not, I can help.

As a follow-up note: always do work on topic branches of your own fork. Never on master.

Comment (philliprower) [inline support/cas-server-support-validation/src/main/java/org/apereo/cas/web/view/Cas30ResponseView.java]: First time I ever merged with a forked project.  I don't think I did it the ideal way either.  I'll try and see if that works merge with master works but I may need to just drop my fork.  Re fork and create a topic branch with this change.  If I need to no problem I don't want to mess up  commit history in your project.

**Comment by dima767** [comment]
Yes, do a fresh fork and a topic branch please and PR from there. I'll close this one. 

Thank you.

**Pull request closed without merging**